### ISSUE TITLE
[GRAPHICAL MENU] Translation of the in game menus

### ISSUE THREAD

**Issue by XCOM-HUB** [vcmi-mods/swedish-translation#9]
This is for @Fabrice-TIERCELIN

 - IN ENGLISH - IN SWEDISH

 - MAIN MENU - HUVUDMENY
 - QUIT TO DESKTOP - AVSLUTA TILL SKRIVBORDET

 - NEW GAME - NYTT SPEL
 - SAVE GAME - SPARA SPEL
 - LOAD GAME - LADDA SPEL
 - RETURN TO GAME - ÅTERGÅ TILL SPEL

 - SINGLE SCENARIO - ENSKILT SCENARIO
 - RESTART SCENARIO - SPELA OM SCENARIO

 - MULTI-PLAYER - ~~FLERA SPELARE~~ FLERSPELARSPEL
 - CAMPAIGN - KAMPANJ
 - TUTORIAL - ~~ÖVNING~~ HANDLEDNING
 - CUSTOM - EGET

 - HIGH SCORE - POÄNGLISTA
 - RESET - ÅTERSTÄLL
 - BACK - TILLBAKA
(Words like STANDARD and EXIT can be the same in Swedish)

 - General - Allmänt
 - Adventure Map - Äventyrskarta
 - Battle - Strid

Let me know if I have forgotten anything.

**Comment by XCOM-HUB** [comment]
I forgot 2 buttons: 

 - SAVE - SPARA
 - LOAD - LADDA

**Comment by XCOM-HUB** [comment]
QUIT = AVSLUTA

SHADOW OF DEATH = DÖDENS SKUGGA
RESTORATION OF ERATHIA = ÅTERUPPRÄTTANDET AV ERATHIA
ARMAGEDDON'S BLADE = HARMAGEDDONS KLINGA
CUSTOM = EGET

HORN OF THE ABYSS = AVGRUNDENS HORN
IN THE WAKE OF GODS = I GUDARS KÖLVATTEN
HEROES CHRONICLES = HJÄLTARNAS KRÖNIKOR

Please see https://github.com/vcmi-mods/h3-themes/pull/9

**Comment by XCOM-HUB** [comment]
CREDITS = MEDVERKANDE

**Comment by Fabrice-TIERCELIN** [comment]
@XCOM-HUB, can you please tell me the translation of the title of [this screen](https://github.com/vcmi-mods/h3-for-vcmi-englisation/blob/vcmi-1.6/H3forVCMIenglisation/content/data/CAMPBACK.png)? It should be easy to render.

**Comment by XCOM-HUB** [comment]
> [@XCOM-HUB](https://github.com/XCOM-HUB), can you please tell me the translation of the title of [this screen](https://github.com/vcmi-mods/h3-for-vcmi-englisation/blob/vcmi-1.6/H3forVCMIenglisation/content/data/CAMPBACK.png)? It should be easy to render.

SELECT A CAMPAIGN = VÄLJ EN KAMPANJ

**Comment by Fabrice-TIERCELIN** [comment]
Here is the file:

<img width="800" height="600" alt="Image" src="https://github.com/user-attachments/assets/aee1102f-16c3-4981-83ee-28439f370daf" />

You can already test it by adding the image in `...\vcmi\Mods\swedish-translation\content\data`. The "CANCEL" word at the bottom will be covered. The AI hasn't been able to render the accent correctly so I draw it manually. Does it render well? I will add this file in a further PR.

- IA: [FLUX.1 Kontext [dev]](https://huggingface.co/spaces/black-forest-labs/FLUX.1-Kontext-Dev)
- Prompt: `Replace the text "SELECT A CAMPAIGN" by "VÄLJ EN KAMPANJ"`

**Comment by XCOM-HUB** [comment]
I have added the image in my DATA folder. It displays correctly for the first two original campaigns, and for Heroes Chronicles. It does not work for the Shadow of Death campaign though.

**Comment by GeorgeK1ng** [comment]
> I have added the image in my DATA folder. It displays correctly for the first two original campaigns, and for Heroes Chronicles. It does not work for the Shadow of Death campaign though.

Maybe SOD uses - campbkx2.png instead?

**Comment by XCOM-HUB** [comment]
> Maybe SOD uses - campbkx2.png instead?

I don’t know. Maybe @Fabrice-TIERCELIN knows!?

**Comment by GeorgeK1ng** [comment]
Yes, it uses campbkx2 - https://github.com/vcmi/vcmi/blob/9a95f249fad50aa7311a1c7d3eeed4aa1547d6b1/config/campaignSets.json#L37

**Comment by Fabrice-TIERCELIN** [comment]
Now I know and I will add it

**Comment by Fabrice-TIERCELIN** [comment]
@XCOM-HUB, what should be written in place of _NEXT CREATURE_ and _START COMBAT_? At this point of the game, it's the tactic step and you can change the location of each creature before the begining of the battle.

- `NEXT CREATURE` allows you to move another creature.
- `START COMBAT` allows you to finish the tactic step and start the battle step.

<img width="1920" height="1032" alt="image" src="https://github.com/user-attachments/assets/80fb9bf2-671e-43f5-95b1-33d31ce2b376" />

**Comment by XCOM-HUB** [comment]
@Fabrice-TIERCELIN 

NEXT CREATURE – NÄSTA VARELSE
START COMBAT – STARTA STRID

I have counted the letters and they should fit perfectly 👌

**Comment by Fabrice-TIERCELIN** [comment]
OK, should I suggest this improvement to Google Translation? [Google Translation suggest me to translate "START COMBAT" "STARTA STRIDEN"](https://translate.google.com/?sl=en&tl=sv&text=NEXT%20CREATURE%0A%0ASTART%20COMBAT&op=translate)

**Comment by XCOM-HUB** [comment]
@Fabrice-TIERCELIN - I chose “STARTA STRID” instead of “STARTA STRIDEN” because it’s shorter and matches the English “START COMBAT” in letter count. Both options work. If “STARTA STRIDEN” fits within the button’s space, feel free to use it.
